File: SearchAndFindBackend/SearchAndFind.DTO/Review/ReviewRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SearchAndFind.DTO
{
    public class ReviewRequest : Request
    {
        public int Points { get; set; }
        public string TenderId { get; set; }
        
    }
}

File: SearchAndFind-Frontend/app/src/main/java/uy/com/searchandfind/review/ReviewRequest.java
package uy.com.searchandfind.review;

import uy.com.searchandfind.Request;

/**
 * Created by Luis on 5/11/2017.
 */

public class ReviewRequest extends Request {

    private int Points;
    private String TenderId;

    public int getPoints() {
        return Points;
    }

    public void setPoints(int points) {
        Points = points;
    }

    public String getTenderId() {
        return TenderId;
    }

    public void setTenderId(String tenderId) {
        TenderId = tenderId;
    }


}
